### PR TITLE
[WIP] Developing

### PR DESCRIPTION
**Pull request by hdthuan**

### CODE REVIEW

File: src/exchange/binance_margin.js [hunk 1]
```diff
@@ -452,6 +527,7 @@ module.exports = class BinanceMargin {
       return;
     }
 
+    this.logger.debug(`TODO: REMOVE: Binance Margin: sync orders: ${JSON.stringify(openOrders)}`);
```
Comment (ArnisL) [inline src/exchange/binance_margin.js:530]: yuck

File: src/exchange/binance_margin.js [hunk 2]
```diff
@@ -440,6 +477,44 @@ module.exports = class BinanceMargin {
     }
   }
 
+  static createOrderBody(order) {
```
Comment (ArnisL) [inline src/exchange/binance_margin.js:480]: I'm out of the loop. How come this is necessary?

File: src/modules/strategy/strategy_manager.js [hunk 3]
```diff
@@ -224,7 +224,7 @@ module.exports = class StrategyManager {
         // check if candle to close time is outside our allow time window
         if (
           validateLookbacks &&
-          !this.technicalAnalysisValidator.isValidCandleStickLookback(lookbacks[exchange].slice(), period)
+          !this.technicalAnalysisValidator.isValidCandleStickLookback(lookbacks[exchange].slice(), period, this.logger)
```
Comment (ArnisL) [inline src/modules/strategy/strategy_manager.js:227]: Well... I'm not up to date with made architectural decisions and whatnot but generally it's a bad idea to pass logger instance in such fashion. Adds way too much noise.

**Comment by hdthuan** [comment]
Oops sorry @ArnisL .

I created this PR by a misclick. I closed it immediately. So this PR is draft.

**Comment by ArnisL** [comment]
There's nothing wrong with drafts.

**Pull request closed without merging**